### ISSUE TITLE
error with @nexus/schema and apollo

### ISSUE THREAD

**Issue by asaadawey** [Errorname/prisma-multi-tenant#63]
i can't get prisma multi tenant to work with @nexus/schema

the `@nexus/schema` came with prisma/cli out of the box 
But the prisma-multi-tenant requires prisma/cli and here is the issue

When @prisma/cli installed as dependancy . i got the following error

```
Error: You most likely forgot to initialize the Prisma Client. Please run `prisma generate` and try to run it again.
If that does not solve your problem, please open an issue.
    at Object.exports.getPrismaClientDmmf (D:\7SSCarRentalProject2\CarRental-Backend\node_modules\nexus-plugin-prisma\src\schema\dmmf\utils.ts:16:11)
    at Object.exports.getTransformedDmmf (D:\7SSCarRentalProject2\CarRental-Backend\node_modules\nexus-plugin-prisma\src\schema\dmmf\transformer.ts:16:47)
    at new SchemaBuilder (D:\7SSCarRentalProject2\CarRental-Backend\node_modules\nexus-plugin-prisma\src\schema\builder.ts:257:7)
    at Object.build (D:\7SSCarRentalProject2\CarRental-Backend\node_modules\nexus-plugin-prisma\src\schema\builder.ts:176:19)
    at Object.onInstall (D:\7SSCarRentalProject2\CarRental-Backend\node_modules\nexus-plugin-prisma\src\schema\index.ts:46:62)
    at D:\7SSCarRentalProject2\CarRental-Backend\node_modules\@nexus\schema\src\builder.ts:657:44
    at Array.forEach (<anonymous>)
    at SchemaBuilder.beforeWalkTypes (D:\7SSCarRentalProject2\CarRental-Backend\node_modules\@nexus\schema\src\builder.ts:651:18)
    at SchemaBuilder.getFinalTypeMap (D:\7SSCarRentalProject2\CarRental-Backend\node_modules\@nexus\schema\src\builder.ts:741:10)
    at makeSchemaInternal (D:\7SSCarRentalProject2\CarRental-Backend\node_modules\@nexus\schema\src\builder.ts:1461:92)
[ERROR] 14:59:16 Error: You most likely forgot to initialize the Prisma Client. Please run `prisma generate` and try to run it again.
If that does not solve your problem, please open an issue.
```

When i remove @prisma/cli i got the following error in runtime

`        "message": "Context creation failed: \n\u001b[31mInvalid \u001b[1m`prisma.tenant.findOne()`\u001b[22m invocation:\u001b[39m\n\n\n  Error in connector: Error querying the database: db error: ERROR: zero-length delimited identifier at or near \"\"\"\"",
`

**Comment by asaadawey** [comment]
I think here is the main problem

```
  Error in connector: Error querying the database: db error: ERROR: zero-length delimited identifier at or near """"
    at PrismaClientFetcher.request (D:\7SSCarRentalProject\NewCode\Backend\node_modules\.prisma-multi-tenant\management\runtime\index.js:1:228459)
```

**Comment by dlehmhus** [comment]
@asaadawey i had the same problem. A dirty fix for me was removing the `@prisma` folder from `@nexus/schema` node_modules folder before generating the nexus prisma schema.

**Comment by Pranit-Harekar** [comment]
I have the same error. Here's my package.json

```json
{
  "scripts": {
    "start": "node dist/server",
    "dev": "ts-node-dev --no-notify --respawn --transpileOnly src/server",
    "clean": "rm -rf dist",
    "build": "npm -s run clean && npm -s run generate && tsc",
    "generate": "npm -s run generate:prisma && npm -s run generate:nexus",
    "generate:prisma": "prisma generate",
    "generate:nexus": "ts-node --transpile-only src/schema"
  },
  "dependencies": {
    "@nexus/schema": "0.16.0",
    "@prisma-multi-tenant/client": "2.3.1",
    "@prisma/client": "^2.9.0",
    "@types/node-fetch": "2.5.7",
    "apollo-server": "2.18.2",
    "graphql": "15.3.0",
    "nexus-plugin-prisma": "^0.22.0"
  },
  "devDependencies": {
    "@types/node": "^14.14.2",
    "prisma-multi-tenant": "^2.3.1",
    "ts-node": "^9.0.0",
    "ts-node-dev": "1.0.0-pre.50",
    "typescript": "^4.0.3"
  },
  "engines": {
    "node": ">=10.0.0"
  }
}

```

**Comment by Pranit-Harekar** [comment]
I fixed it by removing `?schema=public` from the end of pg database URL that you specify in .env file

**Comment by Errorname** [comment]
Hi everyone! This was probably related to this issue: #60 that has been fixed with the release of 2.4.1. If you still encounter this issue, please don't hesitate to reopen. Thanks for your patience 🙏
